cmd/cli/node/list.go

```diff
@@ -16,15 +16,17 @@ import (
 
 var defaultColumnGroups = []string{"labels", "capacity"}
 var orderByFields = []string{"id", "type", "available_cpu", "available_memory", "available_disk", "available_gpu", "status"}
-var filterStatusValues = []string{"approved", "pending", "rejected"}
+var filterApprovalValues = []string{"approved", "pending", "rejected"}
+var filterStatusValues = []string{"healthy", "unhealthy", "unknown"}
```

Comment:
i think it should be binary - "connected" or "unconnected" (we could add something more sophisticated like "unhealthy" when we have more health checks

Reply:
Just for the filters, or for the`node list` as well?

Reply:
i think the node list as well - "healthy" and "unhealthy" imply something i don't think we know. We only know if it's connected or disconnected.

Reply:
Went with Connected/Disconnected for now, discussed with Walid and will add another duration in future to mark the point beyond which we don't think the node is coming back.